extensions/rich_text_components/Video/Video.html

```diff
@@ -1,7 +1,35 @@
 <script type="text/ng-template" id="richTextComponent/Video">
   <center>
-    <iframe width="500" height="280"
-            ng-src="<[videoUrl]>" frameborder="0" allowfullscreen tabindex="<[tabIndexVal]>">
-    </iframe>
+    <div class="oppia-noninteractive-video-iframe-container">
+    <!--  -->
+      <iframe class="oppia-noninteractive-video-iframe" ng-src="<[videoUrl]>"
+              frameborder="0" allowfullscreen tabindex="<[tabIndexVal]>"></iframe>
+    </div>
   </center>
 </script>
+
+<style>
+	/* Allows the video to be the full width of the container
+	 * when put inside a collapsible accordion.
+	 */
+    .conversation-skin-tutor-card-top-content {
+      width: 100%;
+    }
+
+    /* The value of 60.9% is derived from the ratio of the height to the width
+     * (280/480). See http://alistapart.com/article/creating-intrinsic-ratios-for-video
+     * for more details.
+     */
+    .oppia-noninteractive-video-iframe-container {
+		position: relative;
+		padding-bottom: 60.9%;
+    }
+
+    .oppia-noninteractive-video-iframe {
+		position: absolute;
```

Comment:
Nits: please alphabetize the attributes; this makes it easier to scan to see if a particular attribute exists.

Also, please use spaces instead of tabs, throughout this file. (You can see how it currently looks [here](https://github.com/oppia/oppia/pull/1539/files#diff-d5f26ea7080b50b9f9055c7b55cc6657R4).)